docs/meetups/07-18-2023.md

```diff
@@ -17,3 +17,24 @@
    - Sm75.
    - device functions. How hard is this to support while Triton frontend traverses AST?
    - remove torch dependencies from the frontend. (it sounds like there is already progress on this but could be worth discussing)
+
+##### Minutes
+Recording link [here](https://drive.google.com/file/d/1uMlIvih_E5FITwPnNHwTYzo-UKqtey2c/view)
+
+1. Backend plans/broader roadmap:
+   - Plan is for major updates to come in the Triton development meetup which will happen mid-September. For major design changes, currently the plan is to not upstream them directly but have a staging state and different backends can be integrated through a plugin mechanism where Triton provides a layer at the Triton IR layer that is generic and other backends can plug into that.
+   - Short term roadmap plans are very focused on things like helper support, improving all FP8 things on Ampere and Hopper support (end of August). After Hopper support lands, priorities will include refactoring codebase to increase maintainability.
+   - Linalg – upstreaming on hold due to limited dev bandwidth. Want to build an ecosystem where others can leverage Linalg like passes developed in their backend.
+   - For now, peak performance on Nvidia GPUs needs Nvidia specific things, but the convergence of programming models for different backends will allow convergence of hardware backend support in Triton.
+2. Documentation:
+   - Triton has included comments in the backend code.
+   - Seek community involvement to improve tutorials, based on new users knowing what is missing.
+   - Signature changes and doc updates may need some review and time.
+   - Thread created in slack for suggestions on areas needing doc updates. Ian Bearman and his team may have bandwidth to update certain documentation.
+3. Discussion channels:
+   - Preferred #dev channel in slack for technical discussions.
+   - Between GitHub and Slack it would be good to post links into places so folks know discussions are happening elsewhere
+4. CI/testing:
+   - Pretty liberal in terms of accepting regression tests and integration tests for new backends. CI is through nightly whls, which build whls for external backends and run a battery of tests and lightweight frontend tests.
+5. Language improvements: Have released tool for location information, would like to hear more about pain points.
+6. Windows Support: Want to support, some work needs to be done.
```

Comment:
Technically not difficult to get a preliminary version. Most of the maintenance burden would come from having to support it when it breaks.